resources/log4j.properties

```diff
@@ -0,0 +1,9 @@
+## This log4j config file is needed since some of the libraries use, obviously, log4j
```

Comment:
Added this in to be able to debug any Kafka client related issues: like kafka cluster connections.